src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java

```diff
@@ -26,4 +29,9 @@ public void check3 (int...a) { // warn
                 break;
         }
     }
+
+    public void check4() throws IOException {
+            try(InputStream ignored = System.in) { // warn
```

Comment:
please use fully qualified names to avoid import statements. They cause useless diff in violations, it is better to avoid such things

Reply:
Changes Done